plugins/copy-to-clipboard/prism-copy-to-clipboard.js

```diff
@@ -35,13 +35,39 @@
 		head.appendChild(script);
 	}
 
+	function getSettings(start, defaults) {
+		var prefix = 'data-prismjs-';
+		var settings = defaults;
+		for (var key in settings) {
+			var attr = prefix + key;
+			var element = start;
+			while (element && !element.hasAttribute(attr)) {
+				element = element.parentElement;
+			}
+			if (element !== null) {
+				var value = element.getAttribute(attr);
+				if (typeof value !== 'undefined') {
+					settings[key] = value;
+				}
+			}
+		}
+		return settings;
+	}
+
 	Prism.plugins.toolbar.registerButton('copy-to-clipboard', function (env) {
+		var defaults = {
+			'copy': 'Copy',
+			'copy-error': 'Press Ctrl+C to copy',
+			'copy-success': 'Copied!',
+			'copy-timeout': 5000
```

Comment:
Do we really need this? 

Also, `getSettings` doesn't parse numbers, so `setTimeout(..., settings['copy-timeout'])` may throw an error since [it's only defined for numbers](https://html.spec.whatwg.org/multipage/webappapis.html#windoworworkerglobalscope-mixin:dom-settimeout). What do we do if the attribute values isn't a valid number?

Maybe let's remove this option for now and add it later if needed.

Reply:
FWIW, in Chrome at least, a numeric string works fine. A non-numeric string will run instantly. That said, I'm ok cutting it as a setting (cuz I don't think it'll need to be manipulated often) but if we decide to keep it, I'd be ok w/ no validation and users would need to make sure they do it right.

Reply:
I'm ok with keeping it.